elftools/elf/elffile.py

```diff
@@ -138,32 +145,47 @@ def get_dwarf_info(self, relocate_dwarf_sections=True):
         # present.
         # Sections that aren't found will be passed as None to DWARFInfo.
         #
+
+        compressed = bool(self.get_section_by_name('.zdebug_info'))
```

Comment:
good point! It used to be, but after shuffling a code a bit I forgot to relocate this line :)